include/rdmini/vandercorput.h

```diff
@@ -0,0 +1,86 @@
+#ifndef VANDERCORPUT_H_
+#define VANDERCORPUT_H_
+
+#include <iostream>
+#include <random>
+#include <limits> 
+
+//-----------------------------------------------------------------------------
+namespace rdmini{
+class Linear_RNG {
+    public:
+        typedef unsigned result_type;
+
+    private:
+        result_type _s  = 0;
+        static constexpr result_type Nmax = std::numeric_limits<unsigned int>::max();
+
+    public:
+        result_type operator()(){ return (_s++)%Nmax; }
+
+        static result_type min() { return 0; }
+        static result_type max() { return Nmax-1; }
+};
+
+
+//-----------------------------------------------------------------------------
+template <class RealType = double > 
+class vdc_uniform_real_distribution {
+    public:
+        typedef RealType result_type;
+
+        // Nested class: param_type
+        class param_type {
+        private:
+            RealType _M_a;
+            RealType _M_b;
+        public: 
```

Comment:
Identifiers starting with an underscore followed by a capital letter (such as `_M_a`) are reserved identifiers for C++ implementations. It would be best to use names following a different convention for denoting private access.